src/main/java/core/basesyntax/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParserServiceImpl implements ParserService {
+    private static final int NUMBER_OF_TITLE_LINES = 1;
+    private static final String TRANSACTION_SEPARATOR = ",";
+    private static final List<String[]> transactionsList = new ArrayList<>();
+
+    @Override
+    public List<String[]> parseInputData(List<String> inputData) {
+        for (String action : inputData.stream().skip(NUMBER_OF_TITLE_LINES).toList()) {

```

Comment:
I would suggest to use variable for inputData.stream().skip(NUMBER_OF_TITLE_LINES).toList() to make your code readable